internal/cluster/validations/validations.go

```diff
@@ -72,11 +76,25 @@ const (
 )
 */
 
-func ValidatePullSecret(secret string) error {
-	_, err := ParsePullSecret(secret)
+func ValidatePullSecret(secret string, username string, authHandler auth.AuthHandler) error {
+	creds, err := ParsePullSecret(secret)
 	if err != nil {
 		return err
 	}
+
+	if authHandler.EnableAuth {
+		r, ok := creds["cloud.openshift.com"]
+		if !ok {
+			return fmt.Errorf("Pull secret does not contain auth for cloud.openshift.com")
```

Comment:
please use `github.com/pkg/errors` for errors.

Reply:
Done